src/gui/PianoView.cpp

```diff
@@ -618,6 +619,9 @@ void PianoView::mouseMoveEvent( QMouseEvent * _me )
  */
 void PianoView::keyPressEvent( QKeyEvent * _ke )
 {
+#ifdef LMMS_BUILD_LINUX
```

Comment:
16.01.2015 22:51, Tres Finocchiaro пишет:
>
> In src/gui/PianoView.cpp 
> <https://github.com/LMMS/lmms/pull/1632#discussion-diff-23092441>:
>
> > @@ -618,6 +619,9 @@ void PianoView::mouseMoveEvent( QMouseEvent * _me )
> >   */
> >  void PianoView::keyPressEvent( QKeyEvent * _ke )
> >  {
> > +#ifdef LMMS_BUILD_LINUX
>
>     To leave the possibility to use another key for something else:-)
>
> Well if we have to hard-code individual key codes to obtain this 
> feature, I'd argue we're doing something wrong here. How about we use 
> the virtual key for shift and move on.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/LMMS/lmms/pull/1632/files#r23092441>.
>
Not exactly. It is automatically determined when assembling LMMS, as in 
the case of QKeyEvent::key() and Qt::key_Shift. Just in case Qt: 
key_Shift it will be entrusted to himself Cutie, otherwise it will look 
something like this: #ifdef LMMS_BUILD_WIN32 and the like.